docs/examples/vector_stores/QdrantIndexDemo.ipynb

```diff
@@ -1,7 +1,6 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.    Settings.embed_model = FastEmbedEmbedding(model_name="BAAI/bge-base-en-v1.5")
</pre><p> </p><p>Let's use the base model if we're going to set a model</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/11768/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>